### PR TITLE
fix: fixing card icons for semantic-ui 3.0.4 

### LINKED ISSUE #3926: Make the app compatible with Semantic-UI-Ember 3.0.4
With upgrading semantic-UI-ember to 3.0.4, All the modals in the web-app are broke.

More information https://github.com/fossasia/open-event-frontend/issues/3901

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #3926 
Screenshot from 2020-01-31 21-27-10

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by maze-runnar** [comment]
> Can you check if this is the only place where these icons are present?

i am working on that,

**Comment by maze-runnar** [comment]
@snitin315 if you can please tell, where ever these icons are present?

**Comment by kushthedude** [comment]
No, I think you didnt read the description, modals needs to be fjxed. There
is bug in semantic ui for icon which is not a major issue

**Comment by iamareebjamal** [comment]
> There is bug in semantic ui for icon which is not a major issue
It is. Can't merge 3.0.4 until icons are fixed

**Comment by kushthedude** [comment]
> > There is bug in semantic ui for icon which is not a major issue
> 
> It is. Can't merge 3.0.4 until icons are fixed

I will make the PR for semantic , icons are not being imported in them. But modals needs to be fixed with 3.0.4.

File: app/templates/components/public/ticket-list.hbs [hunk 1]
```diff
@@ -153,13 +153,14 @@
...
     <div class="ui row stackable grid">
       <div class="ui less padding thirteen wide computer twelve wide tablet column right aligned floated payment icons">
-        <i class="colored american express link icon"></i>
-        <i class="colored credit card alternative link icon"></i>
-        <i class="colored diners club link icon"></i>
-        <i class="colored discover link icon"></i>
-        <i class="colored mastercard link icon"></i>
-        <i class="colored paypal card link icon"></i>
-        <i class="colored visa link icon"></i>
+        <i class="blue cc amex link icon"></i>
```
Comment (kushthedude) [inline app/templates/components/public/ticket-list.hbs]: Also, the icons you used and icons which are being used a whole lot different.

Comment (maze-runnar) [reply app/templates/components/public/ticket-list.hbs]: I take them from https://semantic-ui.com/elements/icon.html#brands, there were no cards in https://semantic-ui.com/elements/icon.html#payments--shopping .

**Comment by maze-runnar** [comment]
> > > There is bug in semantic ui for icon which is not a major issue
> > 
> > 
> > It is. Can't merge 3.0.4 until icons are fixed
> 
> I will make the PR for semantic , icons are not being imported in them. But modals needs to be fixed with 3.0.4.



> > > There is bug in semantic ui for icon which is not a major issue
> > 
> > 
> > It is. Can't merge 3.0.4 until icons are fixed
> 
> I will make the PR for semantic , icons are not being imported in them. But modals needs to be fixed with 3.0.4.

i will work on them too, if you say so :crossed_fingers:

**Comment by iamareebjamal** [comment]
That's broken on their end as well. Nothing to do with us. 3.0.4 is a patch release, only meant for bugfixes. First, they made a patch release with architectural changes and features which should have been a minor release, i.e., 3.1.0 and now 3.0.4 is breaking downstream apps, which means it should have been 4.0.0. So the modal issue is also an issue in semantic-ember

**Comment by iamareebjamal** [comment]
So ironic that semantic ember does not follow semantic versioning...

**Comment by kushthedude** [comment]
> That's broken on their end as well. Nothing to do with us. 3.0.4 is a patch release, only meant for bugfixes. First, they made a patch release with architectural changes and features which should have been a minor release, i.e., 3.1.0 and now 3.0.4 is breaking downstream apps, which means it should have been 4.0.0. So the modal issue is also an issue in semantic-ember

3.1.0 ? It was not released .They are following the semantic versioning ?

**Comment by iamareebjamal** [comment]
That's the point. 3.0.4 should have not had any new features and definitely not have any breaking change. They should have released 3.1.0 instead of 3.0.4 because it contains new "features". And even then there should have been no breaking change

**Comment by kushthedude** [comment]
Issue in SemanticUI!

**Pull request closed without merging**